packages/ui-menu/src/Menu/__tests__/Menu.test.tsx

```diff
@@ -334,10 +334,10 @@ describe('<Menu />', async () => {
 
     it('should apply offset values to Popover', async () => {
       const getTransforms = (transform: string) => {
-        const transformValues = transform.match(/([\d])\w*/g)!.slice(-2)
+        const transformValues = new DOMMatrixReadOnly(transform)
         return {
-          transformX: parseInt(transformValues[0]),
-          transformY: parseInt(transformValues[1])
+          transformX: Math.floor(transformValues.m41),
+          transformY: Math.floor(transformValues.m42)
```

Comment:
This test had 2 issues:
- The regexp was wrong non fractional values (e.g. it was parsing 5, 6.4 as [6,4]. This issue was hidden by:
- For some reason on local machines the transform's Y value is higher by 0.5 pixel compared to the value in CI. I guess it's some low level Chrome bug, the only difference between the 2 systems is that CI is running on Linux, local dev is OSX I've gone back until 8.14 and could reproduce the issue... Added a Math.floor to ignore subpixel values